spring-kafka-3/src/main/java/com/baeldung/spring/kafka/start/stop/consumer/UserEvent.java

```diff
@@ -0,0 +1,23 @@
+package com.baeldung.spring.kafka.start.stop.consumer;
+
+public class UserEvent {
+
+    private String userEventId;
+
+
+    public UserEvent() {
```

Comment:
If not needed, we should remove this default constructor

Reply:
Need default constructor. 
Caused by: com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot construct instance of `com.baeldung.spring.kafka.start.stop.consumer.UserEvent`